radix-engine-interface/src/api/actor_index_api.rs

```diff
@@ -18,7 +21,12 @@ pub trait ClientActorIndexApi<E> {
     ) -> Result<(), E>;
 
     /// Inserts an entry into an index
-    fn actor_index_insert_typed<K: ScryptoEncode, V: ScryptoEncode>(
+    fn actor_index_insert_typed<
+        // TODO: add a IndexKeyPayloadMarker bound once all native blueprints have been updated
+        K: ScryptoEncode,
+        // TODO: add a IndexEntryPayloadMarker bound once all native blueprints have been updated
```

Comment:
Can we have a single TODO for this? Auditors have been mentioning the number of TODOs in the report.

Reply:
I actually don't think these TODOs are even correct. Will remove.